src/test/kotlin/uk/gov/justice/digital/hmpps/jobsboard/api/jobs/application/MatchingCandidateJobDetailsRetrieverShould.kt

```diff
@@ -21,16 +23,18 @@ class MatchingCandidateJobDetailsRetrieverShould : TestBase() {
   @InjectMocks
   private lateinit var matchingCandidateJobDetailsRetriever: MatchingCandidateJobDetailsRetriever
 
-  private val expectedJobId = expectedJob.id
+  // private val expectedJobId = expectedJob.id
```

Comment:
Same as my previous comment. This will be part of the following PR in the following days, so it's more efficient to leave it until the next PR is merged.